flow_reorganize_symlinks/bigwig_symlink.cwl

```diff
@@ -0,0 +1,38 @@
+#!usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+requirements:
+  - class: InlineJavascriptRequirement
+
+baseCommand: [ln]
+arguments:
+  - prefix: -s
+    #gc file
+    position: 1
+    valueFrom: $(inputs.PATH[0])/$(inputs.tree[0])/$(inputs.tree[1])/$(inputs.deepPATH_bigwig[0])/$(inputs.genome_fasta_name[0]).gc.bigwig.gz
+    #gaps file
+  - position: 2
+    valueFrom: $(inputs.PATH[0])/$(inputs.tree[0])/$(inputs.tree[1])/$(inputs.deepPATH_bigwig[0])/$(inputs.genome_fasta_name[0]).gaps.bigwig.gz
+    #target directory
+  - position: 3
+    valueFrom: $(inputs.MAIN_PATH)/$(inputs.LINK_DIR)/5.Other files/Gaps in Assembly/.
```

Comment:
@HsiuKangHuang the file `*gc.bigwig.gz` needs go into `$(inputs.MAIN_PATH)/$(inputs.LINK_DIR)/5.Other files/GC\ Content/`